subsys/bluetooth/controller/ll_sw/ull_llcp_pdu.c

```diff
@@ -727,27 +727,46 @@ void llcp_pdu_encode_cte_req(struct proc_ctx *ctx, struct pdu_data *pdu)
 	p->cte_type_req = ctx->data.cte_req.type;
 }
 
-void llcp_ntf_encode_cte_req(struct ll_conn *conn, struct pdu_data *pdu)
+void llcp_pdu_decode_cte_rsp(struct proc_ctx *ctx, const struct pdu_data *pdu)
+{
+	if (pdu->cp == 0U || pdu->cte_info.time == 0U) {
+		ctx->data.cte_remote_rsp.has_cte = false;
+	} else {
+		ctx->data.cte_remote_rsp.has_cte = true;
+	}
+}
+
+void llcp_ntf_encode_cte_req(struct pdu_data *pdu)
 {
 	pdu->ll_id = PDU_DATA_LLID_CTRL;
 	pdu->len =
 		offsetof(struct pdu_data_llctrl, cte_rsp) + sizeof(struct pdu_data_llctrl_cte_rsp);
 	pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_CTE_RSP;
 
-	/* TODO add handling of IQ samples forwarding */
+	/* Received LL_CTE_RSP PDU didn't have CTE */
+	pdu->cp = 0U;
```

Comment:
I think earlier you used `true`/`false` for a `uint8_t:1`. How come you are using `0U` and `1U` for this one then?

Reply:
We had a conversation about it already. The outcome was that I had used `bool` for `cp` bit but then it has been updated by @cvinayak to make it closer to specification. I'm using boolean in other places because they are not related with structures from BT spec. or hardware spec.